### DISCUSSION TITLE
Getting error when installing in windows 10

### DISCUSSION THREAD

**Discussion by lightsout** [beetbox/beets#4874] [Q&A] [answered]
I just installed Python according to the documentation. I keep getting this response.

`C:\Users\5820K>pip install beets
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Collecting beets
  Using cached beets-1.6.0.tar.gz (1.7 MB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\python27\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'c:\\users\\5820k\\appdata\\local\\temp\\pip-install-pt0tjc\\beets\\setup.py'"'"'; __file__='"'"'c:\\users\\5820k\\appdata\\local\\temp\\pip-install-pt0tjc\\beets\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'c:\users\5820k\appdata\local\temp\pip-pip-egg-info-r6dscw'
         cwd: c:\users\5820k\appdata\local\temp\pip-install-pt0tjc\beets\
    Complete output (6 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "c:\users\5820k\appdata\local\temp\pip-install-pt0tjc\beets\setup.py", line 38
        print("Could not build manpages (make man failed)!", file=sys.stderr)
                                                                 ^
    SyntaxError: invalid syntax
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.`

**Comment by jackwilsdon** [comment, marked as answer]
Beets no longer supports Python 2 - what documentation did you follow? You need to install Python 3.

**Reply by lightsout** [reply]
Oh great thanks. I followed the link here https://beets.readthedocs.io/en/stable/guides/main.html
Somehow it downloaded v.2, I must have not been paying enough attention, thank you.

**Reply by lightsout** [reply]
Looks like I already had python installed, now I have two versions installed, and pip is trying to use the old version. Even if I run the command from the folder of the new python install I still get the same error.

**Reply by jackwilsdon** [reply]
Try `pip3` instead of `pip`, or uninstall Python 2.7 completely.

**Reply by lightsout** [reply]
Ending up getting it installed thanks.
